systemtest/src/test/java/io/strimzi/systemtest/cruisecontrol/CruiseControlConfigurationST.java

```diff
@@ -92,11 +91,11 @@ void testDeployAndUnDeployCruiseControl() throws IOException {
         LOGGER.info("Verifying that there is no configuration to CruiseControl metric reporter in Kafka ConfigMap");
         assertThrows(WaitException.class, () -> CruiseControlUtils.verifyCruiseControlMetricReporterConfigurationInKafkaConfigMapIsPresent(CruiseControlUtils.getKafkaCruiseControlMetricsReporterConfiguration(testStorage.getNamespaceName(), testStorage.getClusterName())));
 
-        // https://github.com/strimzi/strimzi-kafka-operator/issues/8864
-        if (!Environment.isKRaftModeEnabled() && !Environment.isUnidirectionalTopicOperatorEnabled()) {
+        // TODO https://github.com/strimzi/strimzi-kafka-operator/issues/8864
+        /*if (!Environment.isKRaftModeEnabled()) {
```

Comment:
I guess we should remove this code, rather than commenting it out?

Reply:
I'm keeping this code and TODO comment just because it needs to be addressed as part #8864, which includes other tests.